### PR TITLE
Feature/form errors

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

File: cla_public/templates/base.html [hunk 1]
```diff
@@ -68,14 +68,35 @@
...
       {{ _('Close') }}
     </button>
   </script>
+
+  <script type="text/html" id="mainFormError">
+    <div class="alert alert-error" role="alert">
+      <span class="icon icon-cross" aria-hidden="true"></span>
+      <div class="alert-message">
+        <p>This form has errors.</p>
+        <p>Please see below for the errors you need to correct.</p>
```
Comment (ushkarev) [inline cla_public/templates/base.html]: shouldn't this bit use translation tags?

Comment (s-block) [inline cla_public/templates/base.html]: correct. will do.

**Comment by tyom** [comment]
Excellent work! :+1:

**Comment by tyom** [comment]
retest this please

**Pull request merged**